### ISSUE TITLE
`indent`: new option `offsetTernaryExpressionsOffsetFunctionCalls` breaks linter

### ISSUE THREAD

**Issue by jimmy-zhening-luo** [eslint-stylistic/eslint-stylistic#637]
### Validations

- [X] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

1. Enable `indent` with option `offsetTernaryExpressions: true` (OK).
2. Add another option `offsetTernaryExpressionsOffsetFunctionCalls`, which was added in [v2.12.1](https://github.com/eslint-stylistic/eslint-stylistic/releases/tag/v2.12.1) with #636.
3. Try to lint something.

Expected: Lints.

Actual:

```bash
Oops! Something went wrong! :(

ESLint: 9.16.0

Error: Key "rules": Key "@stylistic/indent":
        Value {"ignoredNodes":[],"SwitchCase":1,"VariableDeclarator":{"var":0,"let":0,"const":0},"outerIIFEBody":0,"MemberExpression":1,"FunctionDeclaration":{"parameters":1,"body":1},"FunctionExpression":{"parameters":1,"body":1},"StaticBlock":{"body":1},"CallExpression":{"arguments":1},"ArrayExpression":1,"ObjectExpression":1,"ImportDeclaration":1,"flatTernaryExpressions":false,"offsetTernaryExpressions":true,"offsetTernaryExpressionsOffsetFunctionCalls":true,"ignoreComments":false,"offsetTernaryExpressionsOffsetCallExpressions":true,"tabLength":4} should NOT have additional properties.
                Unexpected property "offsetTernaryExpressionsOffsetFunctionCalls". Expected properties: "SwitchCase", "VariableDeclarator", "outerIIFEBody", "MemberExpression", "FunctionDeclaration", "FunctionExpression", "StaticBlock", "CallExpression", "ArrayExpression", "ObjectExpression", "ImportDeclaration", "flatTernaryExpressions", "offsetTernaryExpressions", "offsetTernaryExpressionsOffsetCallExpressions", "ignoredNodes", "ignoreComments", "tabLength".

    at RuleValidator.validate (C:\Users\zluo\code\linted\node_modules\eslint\lib\config\rule-validator.js:170:27)
    at new Config (C:\Users\zluo\code\linted\node_modules\eslint\lib\config\config.js:228:27)
    at [finalizeConfig] (C:\Users\zluo\code\linted\node_modules\eslint\lib\config\flat-config-array.js:216:16)
    at FlatConfigArray.getConfigWithStatus (C:\Users\zluo\code\linted\node_modules\@eslint\config-array\dist\cjs\index.cjs:1178:55)
    at FlatConfigArray.getConfig (C:\Users\zluo\code\linted\node_modules\@eslint\config-array\dist\cjs\index.cjs:1196:15)
    at entryFilter (C:\Users\zluo\code\linted\node_modules\eslint\lib\eslint\eslint-helpers.js:282:40)       
    at async NodeHfs.<anonymous> (file:///C:/Users/zluo/code/linted/node_modules/@humanfs/core/src/hfs.js:574:24)
    at async NodeHfs.walk (file:///C:/Users/zluo/code/linted/node_modules/@humanfs/core/src/hfs.js:614:3)    
    at async globSearch (C:\Users\zluo\code\linted\node_modules\eslint\lib\eslint\eslint-helpers.js:323:26)  
    at async Promise.allSettled (index 0)
```

### Reproduction

https://stackblitz.com/edit/github-v7sxzfuc

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

**Comment by jimmy-zhening-luo** [comment]
WHOA nevermind, I just copied and pasted the option name wrong from the documentation. Sorry for the confusion.

**Comment by 9romise** [comment]
Thanks for the feedback. There is indeed a wrong name in the doc, I have send a PR here. #638
